src/par_iter/splitter.rs

```diff
@@ -0,0 +1,66 @@
+use super::internal::*;
+use super::*;
+
+/// The `split` function takes arbitrary data and a closure that knows how to
+/// split it, and turns this into a `ParallelIterator`.
```

Comment:
Should we mark this as `features = unstable`? Also, I think the docs could use with an example or two at least :)